mcu_cfg.hjson

```diff
@@ -136,57 +136,59 @@
     },
 
     interrupts: {
-        // First one is always zero
-        null_intr:               0,
-        uart_intr_tx_watermark:  1,
-        uart_intr_rx_watermark:  2,
-        uart_intr_tx_empty:      3,
-        uart_intr_rx_overflow:   4,
-        uart_intr_rx_frame_err:  5,
-        uart_intr_rx_break_err:  6,
-        uart_intr_rx_timeout:    7,
-        uart_intr_rx_parity_err: 8,
-        gpio_intr_8:             9,
-        gpio_intr_9:             10,
-        gpio_intr_10:            11,
-        gpio_intr_11:            12,
-        gpio_intr_12:            13,
-        gpio_intr_13:            14,
-        gpio_intr_14:            15,
-        gpio_intr_15:            16,
-        gpio_intr_16:            17,
-        gpio_intr_17:            18,
-        gpio_intr_18:            19,
-        gpio_intr_19:            20,
-        gpio_intr_20:            21,
-        gpio_intr_21:            22,
-        gpio_intr_22:            23,
-        gpio_intr_23:            24,
-        gpio_intr_24:            25,
-        gpio_intr_25:            26,
-        gpio_intr_26:            27,
-        gpio_intr_27:            28,
-        gpio_intr_28:            29,
-        gpio_intr_29:            30,
-        gpio_intr_30:            31,
-        gpio_intr_31:            32,
-        intr_fmt_watermark:      33,
-        intr_rx_watermark:       34,
-        intr_fmt_overflow:       35,
-        intr_rx_overflow:        36,
-        intr_nak:                37,
-        intr_scl_interference:   38,
-        intr_sda_interference:   39,
-        intr_stretch_timeout:    40,
-        intr_sda_unstable:       41,
-        intr_trans_complete:     42,
-        intr_tx_empty:           43,
-        intr_tx_nonempty:        44,
-        intr_tx_overflow:        45,
-        intr_acq_overflow:       46,
-        intr_ack_stop:           47,
-        intr_host_timeout:       48,
-        spi2_intr_event:         49,
-        ext_intr:                50:63,
+        number: 64,
```

Comment:
can u pls add a comment that this number should not be changed? I hope people do not change it to 32 or 128

Reply:
I could move it into the `mcu_gen.py` and double check all that there are no duplicates. What do you think?

Reply:
it can stay there that's fine :) it is easier to see thanks